File: PTPREFI.py
import pandas as pd
import matplotlib.pyplot as plt

# Load the dataset
df = pd.read_csv('Medicaldataset.csv')

# Create a histogram for 'age'
plt.figure(figsize=(8, 6))
plt.hist(df['Age'], bins=10, color='skyblue', edgecolor='black')
plt.title('Heart Attack Patients Age Distribution')
plt.xlabel('Age')
plt.ylabel('Number of Patients')
plt.grid(True)
plt.show()

File: PTPREFI.r
library(tidyverse)

# Load the dataset
df <- read_csv('Medicaldataset.csv')
setwd("C:/Users/amiohw/OneDrive - STI College Pasay-Edsa/Desktop/code MS")

# Create a histogram for 'Age'
ggplot(df, aes(x = Age)) +
  geom_histogram(binwidth = 5, fill = 'skyblue', color = 'black') +
  labs(title = 'Heart Attack Patients Age Distribution',
       x = 'Age',
       y = 'Number of Patients') +
  theme_minimal()

